js/flightlog_fields_presenter.js

```diff
@@ -999,8 +1027,12 @@ function FlightLogFieldPresenter() {
                     }
                 case 'RTH':
                     switch(fieldName) {
+                        case 'debug[0]':
+                            return value.toFixed(0) + 'us';
                         case 'debug[1]':
                             return (value / 100).toFixed(1) + 'deg';
+                        case 'debug[2]':
+                            return value.toFixed(0) + 'us';
                         default:
                             return value.toFixed(0);
                     }
```

Comment:
Is better to have them in different order to not duplicate code:
```suggestion
                    switch(fieldName) {
                        case 'debug[0]':
                        case 'debug[2]':                        
                            return value.toFixed(0) + 'us';
                        case 'debug[1]':
                            return (value / 100).toFixed(1) + 'deg';
                        default:
                            return value.toFixed(0);
                    }
```

Reply:
Have revised these and hopefully is OK now.